frontend/src/components/Game.vue

```diff
@@ -58,21 +58,14 @@ export default {
   },
   methods: {
     gameReq: function() {
-      return (
-        "http://backend:3000/games/" +
-        this.$route.params.gameID +
-        "/player/" +
-        this.$route.params.playerID
-      );
+      const gameID = this.$route.params.gameID;
+      const playerID = this.$route.params.playerID;
+      return `http://backend:3000/games/${gameID}/player/${playerID}`;
     },
     updateGame: async function() {
-      try {
-        const req = this.gameReq();
```

Comment:
Why removing the try/catch? Or why was it here the first time?

Reply:
it was there to catch the exception possibly thrown by a connection error in axios.get... but then I changed the catch to just rethrow the error... and then I realized, if I'm just rethrowing the error, why is the `try` even there?